### PR TITLE
Added try catch as per doc

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Added try catch in peer_ops as per the doc design

### CODE REVIEW

**Comment by srijan-sivakumar** [comment]
@aujjwal-redhat , always add description to a PR as it'd be easier for posterity to analyze what the PRs about.

**Review by srijan-sivakumar — APPROVED** [review]
LGTM

File: redant_libs/gluster_libs/peer_ops.py [hunk 1]
```diff
@@ -34,23 +29,30 @@ def peer_probe(server, node):
     node: The node in the cluster where peer probe is to be run
     server: The server to probe
     """
+    try:
 
+        RR.rlogger.info("Redant test framework started")
+        cmd = 'gluster --xml peer probe '+server
```
Comment (nik-redhat) [inline redant_libs/gluster_libs/peer_ops.py:35]: Prefer to use format specifiers.

File: redant_libs/gluster_libs/peer_ops.py [hunk 2]
```diff
@@ -34,23 +29,30 @@ def peer_probe(server, node):
...
 
-    RR.rlogger.info("Redant test framework started")
-    cmd = 'gluster --xml peer probe '+server
+        #TODO: remove the print 
+        print("Running ",cmd," on node ", node)
 
-    #TODO: remove the print 
-    print("Running ",cmd," on node ", node)
+        RR.rlogger.info("Running "+cmd+" on node "+node)
```
Comment (nik-redhat) [inline redant_libs/gluster_libs/peer_ops.py:40]: Same as above

**Comment by aujjwal-redhat** [comment]
I will add the format specifiers in the other patch @nik-redhat

**Pull request merged**